supervisor/homeassistant/core.py

```diff
@@ -125,16 +126,18 @@ async def install_landingpage(self) -> None:
                 await self.instance.install(
                     LANDINGPAGE, image=self.sys_updater.image_homeassistant
                 )
+                break
             except DockerError:
-                _LOGGER.warning("Fails install landingpage, retry after 30sec")
-                await asyncio.sleep(30)
+                pass
             except Exception as err:  # pylint: disable=broad-except
-                self.sys_capture_exception(err)
-            else:
-                self.sys_homeassistant.version = LANDINGPAGE
-                self.sys_homeassistant.image = self.sys_updater.image_homeassistant
-                self.sys_homeassistant.save_data()
-                break
+                capture_exception(err)
+
+            _LOGGER.warning("Fails install landingpage, retry after 30sec")
+            await asyncio.sleep(30)
+
+        self.sys_homeassistant.version = LANDINGPAGE
+        self.sys_homeassistant.image = self.sys_updater.image_homeassistant
+        self.sys_homeassistant.save_data()
```

Comment:
This wasn't related to this PR. But I realized when writing the tests for this when installing the landing page failed due to a docker error we waited 30 seconds before trying again and we didn't wait at all for other types of errors. We just kept retrying forever without waiting until it stopped happening.

That seemed like a bad problem waiting to happen so I made sure the 30 second wait applied to both error cases. I assumed this was an oversight since we did do a 30 second wait in all error cases for the normal `install` and corrected it. If there's a reason for it, let me know, there wasn't a test around it.